tools/istio-clean-iptables/pkg/cmd/root.go

```diff
@@ -130,6 +156,26 @@ func init() {
 		"Specify the GID of the user for which the redirection is not applied. (same default value as -u param)")
 
 	rootCmd.Flags().Bool(constants.RedirectDNS, dnsCaptureByAgent, "Enable capture of dns traffic by istio-agent")
+
+	rootCmd.Flags().String(constants.OwnerUsersInclude, "",
+		"Comma separated list of users whose [outgoing] traffic is to be redirected to Envoy. "+
+			"The wildcard character \"*\" can be used to configure redirection of traffic from all users. "+
+			"A user can be specified either by name or a numeric UID.")
+
+	rootCmd.Flags().String(constants.OwnerUsersExclude, "",
+		"Comma separated list of users whose [outgoing] traffic is to be excluded from redirection to Envoy. "+
+			"Only applies when traffic from all users (i.e. \"*\") is being redirected to Envoy. "+
+			"A user can be specified either by name or a numeric UID.")
+
+	rootCmd.Flags().String(constants.OwnerGroupsInclude, "",
+		"Comma separated list of groups whose [outgoing] traffic is to be redirected to Envoy. "+
```

Comment:
nit: can use ` to do multiline string which can help avoid the need to escape

Reply:
Nit - we should stop adding flags, env variables are sufficient and easier to handle. 
And ProxyConfig too.

We need to start getting rid of the flags we have that duplicate ProxyConfig features too.

Reply:
@costinm I think your comment applies to both commands - `istio-iptables` and `istio-clean-iptables`.

But I'd like to ask a question about `istio-clean-iptables` specifically.

Ideally, I shouldn't be passing settings like a list of UID/GID into `istio-clean-iptables` at all.

As a user, I would expect that `istio-clean-iptables` should simply delete previously installed Iptables rules, even if I don't remember what parameters I used when I called `istio-iptables` earlier.

But, apparently, `istio-clean-iptables` must be given exactly the same parameters as `istio-iptables`. Otherwise, some Iptables rules might not be deleted.

The only reason why `istio-clean-iptables` needs a list of UID/GID, is because `istio-iptables` command appends rules for redirection of DNS over UDP traffic directly into the built-in `OUTPUT` chain instead of using a user-defined chain, e.g. `ISTIO_OUTPUT`, like it's done for all other inbound and outbound traffic.

Is it ok if I refactor `istio-iptables` command to use a user-defined chain for rules of the DNS over UDP traffic? E.g., as a separate PR.